index.js

```diff
@@ -233,6 +239,30 @@ ZongJi.prototype.start = function(options) {
     self.connection._implyConnect();
     self.connection._protocol._enqueue(new self.binlog(function(error, event){
       if(error) return self.emit('error', error);
+      // Add before `event === undefined || event._filtered === true`, so that all nextPosition can be caught
+      if(event){
+        var changed = false;
+        if(event.getTypeName() == 'Rotate'){
```

Comment:
Here and many other places, `==`, `!=` is used where `===`, `!==` should be used.